ansible/group_vars/all/defaults.yaml

```
@@ -20,9 +20,18 @@ kubernetes_cni_http_source: https://github.com/containernetworking/plugins/relea
 crictl_url: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ crictl_version }}/crictl-v{{ crictl_version }}-linux-amd64.tar.gz
 crictl_sha256: 44d5f550ef3f41f9b53155906e0229ffdbee4b19452b4df540265e29572b899c
 containerd_cri_socket: /run/containerd/containerd.sock
+flatcar_containerd_cri_socket: /run/docker/libcontainerd/docker-containerd.sock
 systemd_prefix: /usr/lib/systemd/site-packages
+flatcar_systemd_prefix: /etc/systemd
 sysusr_prefix: /usr
 sysusrlocal_prefix: /usr/local
+flatcar_sysusr_prefix: /opt/
```

nit: idk if this actually causes any issues, but it was `/opt` before this PR.
```suggestion
flatcar_sysusr_prefix: /opt
```